core/src/main/scala/org/apache/spark/scheduler/Stage.scala

```diff
@@ -110,3 +139,8 @@ private[spark] abstract class Stage(
     case _ => false
   }
 }
+
+private[scheduler] object Stage {
+  // The maximum number of times to retry a stage before aborting
```

Comment:
The comment and the variable name don't match here I don't think (since if you tolerate at most 4 failures, it means you retry at most 3 times).  It looks like the variable name is correct so can you update the comment?  Also can you change this to MAX_CONSECUTIVE_FAILURES?